packages/byods/src/types.ts

```diff
@@ -13,6 +15,11 @@ export interface SDKConfig {
    * The client secret of the service app.
    */
   clientSecret: string;
+
+  /**
+   * The token storage adapter passed by the client
+   */
+  tokenStorageAdapter?: TokenStorageAdapter;
```

Comment:
We need to add this as an optional so that users can provide their own implemenations with the config.

Reply:
Make it non-optional since we'll always init it to in-memory adapter